site/smart-accounts/overview.md

```diff
@@ -63,4 +63,4 @@ Transaction batching allows you to bundle multiple transaction calls into a sing
 
 ## 6. Transferring Ownership
 
-Ownership is an important aspect of smart accounts. The Light Account implementation allows you to transfer the ownership of a smart account to another entity. We'll cover this in detail with code examples in the [Transferring Ownership](/guides/transferring-ownership) section.
+Ownership is an important aspect of smart accounts. The LightAccount implementation allows you to transfer the ownership of a smart account to another entity. We'll cover this in detail with code examples in the [Transferring Ownership](/guides/transferring-ownership) section.
```

Comment:
shouldnt this be two words per your changes?

Reply:
yes, thanks for flagging, changed!